samples/tab-navbar-menu/ts/src/components/Tab.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import * as microsoftTeams from '@microsoft/teams-js';
+const navItems = [
+  {
+    id: '1',
+    title: 'Home',
+    icon: "there is an <svg></svg> tag here but I shortened it for easier reading",
+    enabled: true,
+    viewData: null as any,
+    selected: false,
+  },
+  {
+    id: '2',
+    title: 'News',
+    icon: "there is an <svg></svg> tag here but I shortened it for easier reading",
+    enabled: true,
+    viewData: null as any,
+    selected: false,
+  },
+  {
+    id: '3',
+    title: 'Contact',
+    icon: "there is an <svg></svg> tag here but I shortened it for easier reading",
+    enabled: true,
+    viewData: null as any,
+    selected: false,
+  },
+  {
+    id: '4',
+    title: 'About',
+    icon: "there is an <svg></svg> tag here but I shortened it for easier reading",
+    enabled: true,
+    viewData: null as any,
+    selected: false,
+  }
+];
+export interface INavProps {
+}
+interface INavState {
+  menuId: string;
+}
+class TermsOfUse extends React.Component<INavProps, INavState> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      menuId: "0"
+    }
+  }
+  public async componentDidMount() {
+    microsoftTeams.app.initialize();
```

Comment:
app.tsx you have already initialized remove from here, and need to check if it is really needed here.

Reply:
I have updated the changes.